### PR TITLE
Metrics | Weather App

### PR DESCRIPTION
**Pull request by jasonfelice**
### Weather App

- [x] Built `Header`, `Country`, `City`, & `Heading` component.
- [x] Created a Home page with a list of countries that could be filtered by region.
- [x] Created a Cities page that loads a list of cities for the selected country.
- [x] Created a Details page where the weather data loads from the API.
- [x] Styled everything with `scss`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,8 @@
 - webpack
 - React
 
-## Live (Comming Soon)
-[Preview](#)
+## Live
+[Preview](https://venerable-cranachan-299537.netlify.app/)
```
Comment (AmaduKamara) [inline README.md:15]: - [ ] Kindly add a video presenting your project. You are required to record a video on Loom that is at most 5 minutes long explaining the features of the app you built and also a section of your code you are most proud of implementing. Kindly record the video and app the link to the video below this hosted demo link so the next reviewer would be able to watch it.

NOTE: 
- The length of the video is 5 minutes max
- Explain what the project is about
- Explain features of the project by clicking different sections of the app and explaining things
- Lastly, Explain a section of your code you are most proud of.

- [ ] Also you are required to give credit to Nelson Sakwa for his design that was given to you as inspiration. Kindly mention and give credit to the original design creator and the design itself. Here is the link to the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @jasonfelice, 

AlonzoLeroneThisCloseGIF (2)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Good commit messages👍
✅ Good descriptive PR 👍 
✅ App is hosted correctly and works fine 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add a presentation video to your README as instructed in the inline comments below
- [ ] Please give credit to the design creator. He did a lot of work making that design that thousands of students are using as inspiration. He deserves credit. Please do 👍
- [ ] For this capstone project, you are required to write unit tests for your components. Kindly write a test for your project with React testing library.
- 
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by jasonfelice** [comment]
Thanks for the review @AmaduKamara.

**Review by t-yanick — APPROVED** [review]
Hi @jasonfelice, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**